metadata.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "My Awesome App",
```

Comment:
Please use proper name and identifier eg:
"name": "Refapp2"
"identifier": "com.rdkcentral.app.refapp2"

Reply:
metadata.json  update done.